### PR TITLE
Update READMEs for "cmd" binaries

### PR DESCRIPTION
**Pull request by timburks**
This updates, simplifies, and adds some needed details to the READMEs for the binaries in the `cmd` directory.
- `registry-server` configurations are simplified, dropping instructions for running on Cloud Run and GKE. (I expect that we will also remove the related targets from the Makefile)
- The `registry` README consolidates the information in `APIHUB-README.md` (which would be removed and replaced in release builds by `cmd/registry/README.md`). It also includes brief descriptions of a few ways to upload API specs to registries.

### CODE REVIEW

File: cmd/registry/README.md [hunk 1]
```diff
@@ -1,7 +1,112 @@
...
+unquarantine it by running the following on the command line:
+
+```
+xattr -d com.apple.quarantine registry
+```
+
+## Running with Apigee API hub
+
+To use the `registry` tool with a hosted instance associated with Apigee API
+hub, please do the following:
```
Comment (theganyo) [inline cmd/registry/README.md:34]: Link to Apigee API hub?

File: cmd/registry/README.md [hunk 2]
```diff
@@ -1,7 +1,112 @@
...
+
+1. Make sure you have gcloud command installed.
+2. Set the `GCP_PROJECT` environment variable to your API hub project name.
+3. Configure `gcloud` to use your project
```
Comment (theganyo) [inline cmd/registry/README.md:38]: add `:` to end of line

File: cmd/registry/README.md [hunk 3]
```diff
@@ -1,7 +1,112 @@
...
+
+```
+gcloud config set project $GCP_PROJECT
+```
+
+4. Run the following script to get an authorization token and set it in your
+   environment (on Windows, please use `auth/HOSTED.bat`).
```
Comment (theganyo) [inline cmd/registry/README.md:45]: replace `.` with `:` at EOL

Comment (theganyo) [reply cmd/registry/README.md:45]: Also, maybe mention the env variable being set?

File: cmd/registry/README.md [hunk 4]
```diff
@@ -1,7 +1,112 @@
...
+
+```
+registry help
+```
+
+## Importing API information with the Registry tool
+
+The `registry` tool currently includes several demonstration subcommands that
+upload API descriptions into a registry and one subcommmand that is likely to
+become a recommended way to populate an API registry.
```
Comment (theganyo) [inline cmd/registry/README.md:67]: I think having "and one subcommand that is likely..." is ambiguous and confusing.

File: cmd/registry/README.md [hunk 5]
```diff
@@ -1,7 +1,112 @@
...
+
+- `registry upload bulk openapi` reads OpenAPI descriptions from a directory
+  that follows the style of the
+  [APIs-guru/openapi-directory](https://github.com/APIs-guru/openapi-directory)
+  repository. To try it, clone the `openapi-directory` repo, change your
```
Comment (theganyo) [inline cmd/registry/README.md:72]: Include these commands?

File: cmd/registry/README.md [hunk 6]
```diff
@@ -1,7 +1,112 @@
...
+  directory to the repo, and run the following:
+
+  ```
+  registry upload bulk openapi APIs --project-id PROJECT_ID
+  ```
+
+  Here `APIs` is a directory in the repo and `PROJECT_ID` should be replaced
+  with your registry project id.
```
Comment (theganyo) [inline cmd/registry/README.md:80]: Should we ask the user to set this/these as env vars before the command as we do elsewhere?

File: cmd/registry/README.md [hunk 7]
```diff
@@ -1,7 +1,112 @@
...
+  directory that follows the style of the
+  [googleapis/googleapis](https://github.com/googleapis/googleapis) repository.
+  To try it, clone the `googleapis` repo, change your directory to the root of
+  the repo, and run the following:
+
+  ```
+  registry upload bulk protos . --project-id PROJECT_ID
+  ```
+
+  As above, `PROJECT_ID` should be replaced with your registry project id.
```
Comment (theganyo) [inline cmd/registry/README.md:92]: Same comments as above.

File: cmd/registry/README.md [hunk 8]
```diff
@@ -1,7 +1,112 @@
...
+- `registry upload bulk discovery` reads API descriptions from the
+  [Google API Discovery Service](https://developers.google.com/discovery). This
+  reads from an online service, so you can try it by simply running the
+  following:
+
+  ```
+  registry upload bulk discovery --project-id PROJECT_ID
+  ```
+
+  As above, `PROJECT_ID` should be replaced with your registry project id.
```
Comment (theganyo) [inline cmd/registry/README.md:103]: Same comments as above.

File: cmd/registry/README.md [hunk 9]
```diff
@@ -1,7 +1,112 @@
...
+
+- `registry apply` reads API information from YAML files using a mechanism
+  similar to `kubectl apply`. For details, see
+  [this GitHub issue](https://github.com/apigee/registry/issues/450). To try
```
Comment (theganyo) [inline cmd/registry/README.md:107]: Not sure linking to an issue is the best way to document this.

**Review by giteshk — APPROVED** [review]

File: cmd/registry-server/README.md [hunk 10]
```diff
@@ -1,55 +1,58 @@
...
-When that configuration file is specified, the port configuration value can be
-set using the `PORT` environment variable. Other useful environment variables
-are also defined there.
-
-When no configuration is specified, `registry-server` runs on port 8080 using a
-sqlite database stored in a file at `/tmp/registry.db`. For other default
-configuration settings, see
-[cmd/registry-server/main.go](/cmd/registry-server/main.go).
+When no configuration is specified, `registry-server` runs in the evaluation
+mode described in the **Usage** section above.
```
[the remark this answers is not in the document]
Comment (timburks) [reply cmd/registry-server/README.md:28]: Thanks! I didn't know that was possible but found this https://stackoverflow.com/a/15843220

File: cmd/registry-server/README.md [hunk 11]
```diff
@@ -1,43 +1,44 @@
...
 
 ### Running the Registry API server
 
-Run `. auth/LOCAL.sh` to configure your environment to run the Registry API
-server locally and for the included clients to call your local instance. Start
-the server by running `registry-server`.
+To run `registry-server` in its default evaluation mode, simply start it by
+running `registry-server`. Then run `. auth/LOCAL.sh` to set environment
+variables that configure the included clients to call your local instance.
```
[the remark this answers is not in the document]
Comment (timburks) [reply cmd/registry-server/README.md]: That is discussed in the quickstart in the main README. Looking at the context, I think it's better to focus on SQLite usage in this section and updated it accordingly.

File: cmd/registry/README.md (hunk 1 above)
Comment (timburks) [reply cmd/registry/README.md:34]: I'm not aware of any links that we can use now.

File: cmd/registry/README.md (hunk 5 above)
Comment (timburks) [reply cmd/registry/README.md:72]: ok done

File: cmd/registry/README.md (hunk 6 above)
Comment (timburks) [reply cmd/registry/README.md:83]: @giteshk I think we would only want to use GCP_PROJECT for situations where a GCP project is required - that's not the case for instances that aren't GCP-hosted.

File: cmd/registry/README.md (hunk 4 above)
Comment (timburks) [reply cmd/registry/README.md]: thanks, improved slightly by explicitly mentioning `registry apply`, we can reword further as this section develops

File: cmd/registry/README.md (hunk 6 above)
Comment (timburks) [reply cmd/registry/README.md]: Replacing these with $PROJECT_ID consistently for now

File: cmd/registry/README.md (hunk 8 above)
Comment (timburks) [reply cmd/registry/README.md]: fixed

File: cmd/registry/README.md (hunk 7 above)
Comment (timburks) [reply cmd/registry/README.md]: fixed

File: cmd/registry/README.md (hunk 3 above)
Comment (timburks) [reply cmd/registry/README.md]: done (both), thanks

File: cmd/registry/README.md [hunk 12]
```diff
@@ -1,7 +1,112 @@
...
+includes the address of the Registry API server and authentication tokens.
+
+## Release builds
+
+Release builds of the `registry` tool are available on GitHub and can be found
+in
+[github.com/apigee/registry/releases](https://github.com/apigee/registry/releases).
+
+**MacOS note:** To run the `registry` tool on MacOS, you will need to
+unquarantine it by running the following on the command line:
```
[the remark this answers is not in the document]
Comment (timburks) [reply cmd/registry/README.md]: I added a link to an Apple forums site that discusses this. Let them explain it! :)

**Comment by timburks** [comment]
Thanks @theganyo and @giteshk - going with this but happy to take additional feedback (@earth2marsh @seaneganx, feel free to comment in any forum)

**Pull request merged**